SoftLayer/managers/event_log.py

```diff
@@ -0,0 +1,104 @@
+"""
+    SoftLayer.event_log
+    ~~~~~~~~~~~~~~~~~
+    Network Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+from SoftLayer import utils
+
+
+class EventLogManager(object):
+    """Provides an interface for the SoftLayer Event Log Service.
+
+    See product information here:
+    http://sldn.softlayer.com/reference/services/SoftLayer_Event_Log
+    """
+
+    def __init__(self, client):
+        self.event_log = client['Event_Log']
+
+    def get_event_logs(self, request_filter):
```

Comment:
TODO: event_log.getAllObjects  should be called with `self.event_log.getAllObjects(filter=request_filter, iter=True)` and likely needs to support limits so more than the default number of objects can be pulled down.